src/utils/actions.js

```diff
@@ -218,10 +218,22 @@ const actions = [
     },
     requiredInput: ['variable', 'value'],
   },
+  {
+    value: 'doReload',
+    translation: 'Object.ActionButton.DoReload',
+    group: 'reload',
+    getActionCall: ({ app, partial }) => async () => {
+      const e = await app.doReload(0, !!partial, false);
+      if (e) {
```

Comment:
What is e? error?

Reply:
true or false, if the reload succeded or not.
I'm not checking for any permissions etc, so if you try to do a reload without update permissions I assume you'll get an generic "Access denied" message as well.